db/schema.rb

```diff
@@ -10,11 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_12_07_191632) do
+ActiveRecord::Schema.define(version: 2018_12_10_185019) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "books", force: :cascade do |t|
```

Comment:
This came from the next PR should not be here ;)

Reply:
The schema.rb is autogenerated, I have that table in the schema.rb because I've run a new migration on this branch (to removed unused columns from users) and before that I've run a migration on another branch to create table books. How can I solve that? Do I have to manually remove that table from schema.rb?